internal/helpers/helpers_test.go

```diff
@@ -0,0 +1,84 @@
+package helpers
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type testStringer struct{}
+
+func (ts testStringer) String() string {
+	return "stringer"
+}
+
+func TestToString(t *testing.T) {
+	assert.Equal(t, "string", ToString("string"))
+	assert.Equal(t, "5", ToString(5))
+	assert.Equal(t, "true", ToString(true))
+	assert.Equal(t, "bytes", ToString([]byte("bytes")))
+	assert.Equal(t, "error", ToString(errors.New("error")))
+	assert.Equal(t, "stringer", ToString(testStringer{}))
+}
+
+func TestEmpty(t *testing.T) {
```

Comment:
While this test is testing what is empty, it should also test that we get false for 1, .1, true, `" "`, []string{""}